### PR TITLE
Build wheel for python 3.8 and python 3.9

### PR DESCRIPTION
**Pull request by Abhishek-Dev09-zz**
Adding Support for py 37/38/39 for all oses.

Signed-off-by: Abhishek Kumar <abhishek.kasyap09@gmail.com>

### CODE REVIEW

**Comment by Abhishek-Dev09-zz** [comment]
@pombredanne : Kindly review and merge this.

File: .travis.yml [hunk 1]
```diff
@@ -13,22 +13,22 @@ env:
     global:
         - PLAT=x86_64
         - MB_PYTHON_OSX_VER=10.9
-        - PROJECT_SPEC="pyahocorasick pyahocorasick==1.4.0 lxml lxml==4.2.1 simplejson simplejson==3.15.0 intbitset intbitset==2.3.0 bitarray bitarray==0.8.1 pycryptodome pycryptodome==3.6.1 rdflib rdflib==4.2.1 url url==0.1.4.5 url==0.1.6 scandir scandir==1.9.0 MarkupSafe MarkupSafe==1.0 coverage==4.4.1 coverage pdfminer.six pdfminer.six==20181108 pefile pefile==2018.8.8 pyyaml PyYAML==3.12 SPARQLWrapper SPARQLWrapper==1.8.0 psutil psutil==5.6.2 tinyarray cdifflib edlib hyperscan pyroaring"
+        - PROJECT_SPEC="pyahocorasick pyahocorasick==1.4.0 lxml lxml==4.2.1 simplejson simplejson==3.15.0 intbitset intbitset==2.3.0 bitarray bitarray==0.8.1 pycryptodome pycryptodome==3.6.1 rdflib rdflib==4.2.1 urlpy scandir scandir==1.9.0 MarkupSafe MarkupSafe==1.0 coverage==4.4.1 coverage pdfminer.six pdfminer.six==20181108 pefile pefile==2018.8.8 pyyaml PyYAML==3.12 SPARQLWrapper SPARQLWrapper==1.8.0 psutil psutil==5.6.2 tinyarray cdifflib edlib hyperscan pyroaring"
 
     matrix:
         - MB_PYTHON_VERSION=2.7 UNICODE_WIDTH=16
```
Comment (pombredanne) [inline .travis.yml]: May be you can drop 2.7 ?

Comment (Abhishek-Dev09-zz) [reply .travis.yml]: yes

File: .travis.yml [hunk 2]
```diff
@@ -13,22 +13,22 @@ env:
     global:
         - PLAT=x86_64
         - MB_PYTHON_OSX_VER=10.9
-        - PROJECT_SPEC="pyahocorasick pyahocorasick==1.4.0 lxml lxml==4.2.1 simplejson simplejson==3.15.0 intbitset intbitset==2.3.0 bitarray bitarray==0.8.1 pycryptodome pycryptodome==3.6.1 rdflib rdflib==4.2.1 url url==0.1.4.5 url==0.1.6 scandir scandir==1.9.0 MarkupSafe MarkupSafe==1.0 coverage==4.4.1 coverage pdfminer.six pdfminer.six==20181108 pefile pefile==2018.8.8 pyyaml PyYAML==3.12 SPARQLWrapper SPARQLWrapper==1.8.0 psutil psutil==5.6.2 tinyarray cdifflib edlib hyperscan pyroaring"
+        - PROJECT_SPEC="pyahocorasick pyahocorasick==1.4.0 lxml lxml==4.2.1 simplejson simplejson==3.15.0 intbitset intbitset==2.3.0 bitarray bitarray==0.8.1 pycryptodome pycryptodome==3.6.1 rdflib rdflib==4.2.1 urlpy scandir scandir==1.9.0 MarkupSafe MarkupSafe==1.0 coverage==4.4.1 coverage pdfminer.six pdfminer.six==20181108 pefile pefile==2018.8.8 pyyaml PyYAML==3.12 SPARQLWrapper SPARQLWrapper==1.8.0 psutil psutil==5.6.2 tinyarray cdifflib edlib hyperscan pyroaring"
```
Comment (pombredanne) [inline .travis.yml:16]: I wonder if there would be a way to get that easier to maintain than this long line?
... possibly using a plain requirement file of sorts?

File: appveyor.yml [hunk 3]
```diff
@@ -47,6 +46,8 @@ environment:
     - PYTHON: "C:\\Python37-x64"
     - PYTHON: "C:\\Python38"
     - PYTHON: "C:\\Python38-x64"
+    - PYTHON: "C:\\Python39"
```
Comment (pombredanne) [inline appveyor.yml:47]: may be time to drop PYTHON: "C:\\Python27" above?

Comment (Abhishek-Dev09-zz) [reply appveyor.yml:47]: dropped!

File: appveyor.yml [hunk 4]
```diff
@@ -57,7 +58,7 @@ install:
 build: false
 
 test_script:
-  - python -m pip wheel --wheel-dir=wheelhouse url || dir
+  - python -m pip wheel --wheel-dir=wheelhouse urlpy || dir
```
Comment (pombredanne) [inline appveyor.yml:59]: Why not having this one wheel in the same list as above?

Comment (Abhishek-Dev09-zz) [reply appveyor.yml:59]: brcause url is not supported by python 37

**Review by pombredanne — APPROVED** [review]
LGTM... thanks!

**Pull request merged**